### PR TITLE
[W5.11][W12-4] Liu Tianhang

### PR DESCRIPTION
**Pull request by LIU-TIANHANG**
The find function now search for people irregardless of case

### CODE REVIEW

**Comment by Lunastryke** [comment]
Good implementation

**Comment by bluein-green** [comment]
Would it be better if you created a separate feature for case-insensitive search? So that you don't lose out on case-sensitive searches that some users may desire more.

**Comment by CT15** [comment]
Aside from the commit message, this PR looks good to me. Good job! 👍
If you do not have any queries, you can close this PR.

File: src/seedu/addressbook/commands/FindCommand.java [hunk 1]
```diff
@@ -48,13 +48,30 @@ public CommandResult execute() {
...
+            if (!Collections.disjoint(wordsInNameToUpper, keywordUpper)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
 
+    /**
+     *
+     * @param keywords
```
Comment (CT15) [inline src/seedu/addressbook/commands/FindCommand.java:65]: You can refer to other methods' javadoc comment and see how they are written.

File: src/seedu/addressbook/commands/FindCommand.java [hunk 2]
```diff
@@ -48,13 +48,30 @@ public CommandResult execute() {
      */
```
Comment (CT15) [inline src/seedu/addressbook/commands/FindCommand.java:48]: Can consider changing the javadoc comment to reflect your changes.

**Review by CT15 — COMMENTED** [review]
Added minor things about javadoc comment that you may want to pay attention to.

**Pull request closed without merging**